MMTabBarView/MMTabBarView/MMAttachedTabBarButton.m

```diff
@@ -188,22 +188,28 @@ - (NSRect)draggingRect {
     return draggingRect;
 }
 
+inline static NSBitmapImageRep* imageForView(NSView* const inView, NSRect const inBounds) {
+	if (@available(macOS 10.14, *)) {
```

Comment:
Just for readability, we should put the remainder of the non-10.14 code into an else-branch.

Reply:
I also noticed that, but I hesitated as the goal was to reduce differences with @MiMo42 's version.
Changed as you suggested.